token-lending/program/src/instruction.rs

```diff
@@ -31,257 +31,264 @@ pub enum LendingInstruction {
     ///
     /// Accounts expected by this instruction:
     ///
-    ///   0. `[writable]` Lending market account.
-    ///   1. `[]` Quote currency SPL Token mint. Must be initialized.
+    ///   0. `[writable]` Lending market account - uninitialized
+    ///   1. `[]` Quote currency SPL Token mint
     ///   2. `[]` Rent sysvar
-    ///   3. '[]` Token program id
+    ///   3. `[]` Token program id
     InitLendingMarket {
         /// Owner authority which can add new reserves
-        market_owner: Pubkey,
+        owner: Pubkey,
+        /// The target ratio of an obligation's borrows to deposits as a percent
+        loan_to_value_ratio: u8,
```

Comment:
Hmm. How do you think we should determine the LTV and liquidation threshold for an obligation with multiple currencies? I moved this here because that seems conceptually problematic for users to understand (and as far as I can tell, not what Aave does).

Reply:
This is actually what Aave does, you can see how the calculate a user's position by iterating over all the reserves here: https://github.com/aave/aave-protocol/blob/f7ef52000af2964046857da7e5fe01894a51f2ab/contracts/lendingpool/LendingPoolDataProvider.sol#L70

Reply:
Ahhh, interesting. Okay, cool. I'll add this.

Reply:
Done.